modular_skyrat/modules/delam_emergency_stop/code/scram.dm

```diff
@@ -427,27 +427,68 @@
-		- Keep an eye out for fires and the infamous air mix. It's always an adventure trying to strike the perfect balance between breathable air and potential suffocation. Remember, oxygen plus a spark equals fireworks – the kind you definitely don't want inside a reactor.<br><br>\
-		- To avoid singeing your eyebrows off, consider enlisting the help of a synth or a trusty borg. After all, nothing says \"safety first\" like outsourcing your firefighting to non-living, non-breathing assistants.<br><br>\
-		- Clear out any lightly radioactive debris (The cargo department will probably love to dispose it for you.)<br><br>\
-		- Finally, revel in the satisfaction of knowing that you've single-handedly prevented a delamination. But, of course, don't forget to feel guilty because SAFETY MOTH Knows. SAFETY MOTH knows everything. It's always watching, judging, and probably taking notes for its next safety briefing. So bask in the glory of your heroism, but know that the all-knowing Moff is onto you.<br><br>\
-		<b>(Optional step, for the true daredevils out there)</b><br><br>\
-		- When it comes time for your second attempt at starting the SM: Fold these instructions into a paper plane, give it a good toss towards the crystal, and watch it soar through the air. Because nothing says \"I'm dealing with a potentially catastrophic situation\" like engaging in some whimsical paper airplane shenanigans.<br><br>\
-		<b>Hopefully you'll never need to use this. However, good luck!</b>"
 	return ..()
 
+/obj/item/paper/paperslip/corporate/fluff/delam_procedure/examine(mob/user)
+	. = ..()
+	ui_interact(user)
+
+/obj/item/paper/paperslip/corporate/fluff/delam_procedure/attackby(obj/item/attacking_item, mob/living/user, params)
+	if(burn_paper_product_attackby_check(attacking_item, user))
+		SStgui.close_uis(src)
+		return
+
+	// Enable picking paper up by clicking on it with the clipboard or folder
+	if(istype(attacking_item, /obj/item/clipboard) || istype(attacking_item, /obj/item/folder) || istype(attacking_item, /obj/item/paper_bin))
+		attacking_item.attackby(src, user)
+		return
+
+	ui_interact(user)
+	return ..()
+
+/obj/item/paper/paperslip/corporate/fluff/delam_procedure/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "DelamProcedure")
+		ui.open()
+
+/obj/structure/sign/delam_procedure
+	name = "Safety Moth - Delamination Emergency Procedure"
+	desc = "This informational sign uses Safety Moth™ to tell the viewer how to use the emergency stop button if the Supermatter Crystal is delaminating."
+	icon = 'modular_skyrat/modules/delam_emergency_stop/icons/scram.dmi'
+	icon_state = "moff-poster"
+	pixel_y = 4
+	armor_type = /datum/armor/sign_delam
+	anchored = TRUE
+
+/datum/armor/sign_delam
+	melee = 60
+	acid = 70
+	fire = 90
+
+/obj/structure/sign/delam_procedure/examine(mob/user)
+	. = ..()
+	ui_interact(user)
```

Comment:
Might want to make it so it only puts up the UI if the user is like, within a tile or two, maybe? Would make a bit more sense, I reckon.

Reply:
The default ui_state performs the regular checks, it'll only work at most 2 tiles away.

https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/2484cc98-f69f-457a-b829-2f4b314f819f

Reply:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/1f112895-b870-4443-af19-cd819814885a)